src/components/auth/profile/DonationTab.tsx

```diff
@@ -1,205 +1,138 @@
-                        Свали <ArrowForwardIcon />
-                      </Button>
-                    </TableCell>
-                  </TableRow>
-                ))}
-              </TableBody>
-            </Table>
-          </TableContainer>
-        ) : (
-          <Box sx={{ fontSize: 20, mt: 4 }}>Към момента няма направени дарения</Box>
-        )}
-      </Box>
+    <ProfileTab
+      title={user?.user ? user.user.firstName + ' ' + user?.user.lastName : ''}
+      name={ProfileTabs.donations}>
+      <Typography variant="h5" fontWeight={'medium'}>
+        {t('profile:donations.helpThanks')} ❤️
+      </Typography>
+      <Grid
+        container
+        spacing={theme.spacing(2)}
+        marginTop={theme.spacing(1)}
+        alignItems={'flex-end'}>
+        <Grid order={matches ? 3 : 1} item xs={12} md={4}>
+          <Card>
+            {!isCampaignLoading && campaigns ? (
+              <CardActionArea>
+                <CardMedia
+                  component="img"
+                  height="193"
+                  image={campaignListPictureUrl(campaigns[0])}
+                  alt={campaigns[0].title}
+                />
+                <CardContent>
+                  <Typography gutterBottom variant="h5" component="div">
+                    {campaigns[0].title}
+                  </Typography>
+                  <Typography variant="body2" color="text.secondary">
+                    {truncate(campaigns[0].description, { length: 120 })}
+                  </Typography>
+                </CardContent>
+              </CardActionArea>
+            ) : (
+              <CircularProgress />
+            )}
+            <CardActions>
+              <Button variant="contained" size="medium" color="secondary">
+                {t('profile:donations.donateNow')} ❤️
+              </Button>
+            </CardActions>
+          </Card>
+        </Grid>
+        <Grid order={matches ? 1 : 2} item xs={12} md={8}>
+          {!isUserDonationLoading && userDonations ? (
+            <Card className={classes.donationsBox}>
+              <Box className={classes.donationsBoxRow}>
+                <Typography fontWeight="medium" variant="h5">
+                  {t('profile:donations.totalDonations')}
+                </Typography>
+                <Typography fontWeight="medium" variant="h5">
+                  {money(userDonations.total)}
+                </Typography>
+              </Box>
+              <Box className={classes.donationsBoxRow}>
+                <Box>
+                  <Typography variant="h5">{t('profile:donations.recurringDonations')}</Typography>
+                  {/* TODO: Use date-fns to format and localize the months,
+                   that the user has recurring donations when that is possible */}
+                  <Typography>Я, Ф, М, А 2022</Typography>
+                </Box>
+                <Typography fontWeight="medium" variant="h5">
+                  {money(userDonations.donations[0].amount)}
```

Comment:
Accessing donations by index is an assumption that there will always be at least one donation.

This assumption did break staging app today

![image](https://user-images.githubusercontent.com/893608/164486890-040f8912-083e-424a-96a0-3017a54c40e7.png)

Reply:
I will also make it check and display some placeholder when I have the time

Reply:
It's fixed already with https://github.com/podkrepi-bg/frontend/commit/aa1f4498266c9bf579b9127156f0fbde587b0591